pkg/controller/import-controller.go

```diff
@@ -76,6 +84,32 @@ func (ic *ImportController) findImportPodFromCache(pvc *v1.PersistentVolumeClaim
 	return podList[0], nil
 }
 
+func (ic *ImportController) requiresScratchSpace(pvc *v1.PersistentVolumeClaim) bool {
+	contentType := getContentType(pvc)
+	// All archive requires scratch space.
+	if contentType == "archive" {
+		glog.V(3).Infof("Archive content type, requires scratch space\n")
+		return true
+	}
+
+	switch getSource(pvc) {
+	case SourceS3:
+		return true
+	case SourceGlance:
+		return true
+	case SourceRegistry:
+		return true
+	}
+
+	value, ok := pvc.Annotations[AnnRequiresScratch]
+	if ok {
+		boolVal, _ := strconv.ParseBool(value)
+		return boolVal
+	}
+	glog.V(3).Infof("No scratch space\n")
```

Comment:
/nit
"no scratch space needed for import of PVC: %s", pvc.Name

Reply:
/nit
I don't know how detailed we want to be in our logging, but if we want to follow the pattern you have for logging scratch not needed; might conisder collapsing all of the returns into a return value, then before leaving the routine have a single log statements for scratch or !scratch.  So something like:

```
scratchRequired := false
if contenteType == "archive"{
  scratchRquire = true
}
else {
  switch getSource(pvc){
    case SourceS3:
      scratchRequired = true
.......
........
glog.V(3).Infof("import pvc %s scratch requirement is %v", pvc.Name, scratchRequired)
return scratchRequired
```

I think the if blocks and cases could probably be optimized a bit but maybe not, that's not a big deal IMO

Reply:
Well I am expecting some changes to this in the near future as narrow down what needs and what does not need scratch space. I will update the logging.